frontend/src/nc/pages/components/EditModal.js

```diff
@@ -181,8 +181,8 @@ const EditModal = ({ editObj, setEditObj, columns_, refreshFunc }) => {
   };
```

Comment:
should the extension of this file be jsx?

Reply:
Done